client/src/components/Footer.vue

```diff
@@ -70,5 +70,5 @@
 </template>
 
 <script setup lang="ts">
-import FooterBanner from '/images/Footer-min.jpeg'
+import FooterBanner from '../../public/images/Footer-min.webp'
```

Comment:
we shouldnt have to do ../../public
"/images/" will automatically be the public folder. This is the same for all other pages